livre/eq/Channel.cpp

```diff
@@ -382,18 +382,16 @@ struct Channel::Impl
         const DataSource& dataSource =
                 node->getTextureDataCache().getDataSource();
         const VolumeInformation& info = dataSource.getVolumeInfo();
-        Vector3f voxelSize = info.boundingBox.getSize() / info.voxels;
-        std::string unit = "m";
-        if( voxelSize.x() < 0.000001f )
-        {
+        const Vector3f resolution = info.resolution;
+
+        const float ratio = 1.0f / dataSource.getVolumeInfo().meterToDataUnitRatio;
```

Comment:
Don't call getVolumeInfo again, use the local variable info.

In the future getVolumeInfo could be a potentially slow operation (e.g. if a network connection is involved), does it make sense to cache the volume info in a member variable and update it if the URI has changed?